src/Mod/Draft/draftmake/make_label.py

```diff
@@ -280,11 +282,18 @@ def make_label(target_point=App.Vector(0, 0, 0),
     if not custom_text:
         custom_text = "Label"
     try:
-        utils.type_check([(custom_text, str)], name=_name)
+        utils.type_check([(custom_text, (str, list))])
     except TypeError:
-        _err(translate("draft","Wrong input: must be a string."))
+        _err(translate("draft","Wrong input: must be a list of strings or a single string."))
+        return None
+
+    if not all(isinstance(element, str) for element in custom_text):
+        _err(translate("draft","Wrong input: must be a list of strings or a single string."))
         return None
 
+    if isinstance(custom_text, str):
+        custom_text = [custom_text]
```

Comment:
According https://github.com/FreeCAD/FreeCAD/blob/9b2ebec35dca5497b1595d97682768a196ad348b/src/Mod/Draft/draftobjects/label.py#L170 this isn't really needed as CustomText property can also accept a single string. However I didn't perform any run check about that.

Reply:
You are completely correct. Thanks for pointing this out. Note that these lines were copied from make_text where they can also be omitted.

Reply:
I am going to try to update this PR...